src/Ledger/PDF.lagda

```diff
@@ -36,23 +36,26 @@
 \restylefloat{figure}
 
 \usepackage{iohk}
-
+\usepackage{fls-macros}
 \usepackage[hidelinks]{hyperref}
 \usepackage[links]{agda}
 
 \newcommand{\N}{\ensuremath{\mathbb{N}}}
-\newcommand{\TxBody}{\type{TxBody}}
-\newcommand{\TxWitness}{\type{TxWitness}}
-\newcommand{\Tx}{\type{Tx}}
-\newcommand{\Ix}{\type{Ix}}
-\newcommand{\TxId}{\type{TxId}}
-\newcommand{\Addr}{\type{Addr}}
-\newcommand{\UTxO}{\type{UTxO}}
-\newcommand{\Coin}{\type{Coin}}
-\newcommand{\TxIn}{\type{TxIn}}
-\newcommand{\TxOut}{\type{TxOut}}
-\newcommand{\UTxOEnv}{\type{UTxOEnv}}
-\newcommand{\UTxOState}{\ensuremath{\type{UTxOState}}}
+
+%% -- These are now defined in agda-macros.sty --------------------
+%% \newcommand{\TxBody}{\type{TxBody}}
+%% \newcommand{\TxWitness}{\type{TxWitness}}
+%% \newcommand{\Tx}{\type{Tx}}
+%% \newcommand{\Ix}{\type{Ix}}
+%% \newcommand{\TxId}{\type{TxId}}
+%% \newcommand{\Addr}{\type{Addr}}
+%% \newcommand{\UTxO}{\type{UTxO}}
+%% \newcommand{\Coin}{\type{Coin}}
+%% \newcommand{\TxIn}{\type{TxIn}}
+%% \newcommand{\TxOut}{\type{TxOut}}
+%% \newcommand{\UTxOEnv}{\type{UTxOEnv}}
```

Comment:
All of these (or at least the ones that we are actually using) were moved to the `fls-macros.sty` file and their definitions were changed from using the generic command "\type{}" to the appropriate Agda type.